config/containers/container-networking.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Container networking
+description: How networking works from the container's point of view
+keywords: networking, container, standalone
+redirect_from:
+- /engine/userguide/networking/configure-dns/
+- /engine/userguide/networking/default_network/binding/
+---
+
+The type of network a container uses, whether it is a [brudge](bridges.md), an
```

Comment:
s/brudge/bridge/

Reply:
Thanks, this will be fixed in #5945. As this content is now live, please file new issues when you see problems.